dbchanges/reviews.sql

```diff
@@ -0,0 +1,152 @@
+USE foody;
+
+DROP TABLE IF EXISTS foody.restaurants;
+CREATE TABLE foody.restaurants (
+  restaurantId int AUTO_INCREMENT PRIMARY KEY,
+  restaurantName varchar(200),
+  restaurantAddress varchar(200),
+  restaurantPhone varchar(200),
+  restaurantWebsite varchar(200),
+  postDate datetime NOT NULL DEFAULT CURRENT_TIMESTAMP()
+);
+
+
+DROP TABLE IF EXISTS foody.restaurantReviews;
+CREATE TABLE foody.restaurantReviews (
+  reviewId int AUTO_INCREMENT PRIMARY KEY,
+  restaurantId int NOT NULL,
+  userId int NOT NULL,
+  review varchar(500) NOT NULL,
+  deliciousnessScore int,
+  serviceScore int,
+  experienceScore int,
+  pricingScore int,
+  pricingValue float,
+  postDate datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
+  updateDate datetime NOT NULL DEFAULT CURRENT_TIMESTAMP()
+)
```

Comment:
We should always use capital letters with MySQL. Also, we should use underscores for spaces and lowercase formatting. 

DROP TABLE IF EXISTS foody.restaurant_reviews;
CREATE TABLE foody.restaurant_reviews (
  review_id INT(6) AUTO_INCREMENT PRIMARY KEY,
  restaurant_id INT(6) NOT NULL,
  user_id int NOT NULL,
  review_text VARCHAR(500) NOT NULL,
  deliciousness_score INT(6) NOT NULL,,
  service_score INT(6) NOT NULL,,
  experience_score INT(6) NOT NULL,,
  pricing_score INT(6) NOT NULL,,
  pricing_value FLOAT NOT NULL,,
  post_date datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  update_date datetime NOT NULL DEFAULT CURRENT_TIMESTAMP()
)